spec/controllers/admin/users_controller_spec.rb

```diff
@@ -18,6 +18,19 @@
       expect(response).to render_template :index
     end
   end
+  describe 'PATCH #togle_confirmation' do
+    it 'confirms user' do
+      unconfirmed_user = create(:user, email: 'unconfirmed_user@osem.io', confirmed_at: nil)
+      patch :toggle_confirmation, id: unconfirmed_user.id, user: { to_confirm: 'true' }
+      confirmed_user = User.find(unconfirmed_user.id)
```

Comment:
The test keeps on failing if I remove that line. 
```ruby
  describe 'PATCH #toggle_confirmation' do
    it 'confirms user' do
      user = create(:user, email: 'unconfirmed_user@osem.io', confirmed_at: nil)
      patch :toggle_confirmation, id: user.id, user: { to_confirm: 'true' }
      expect(user.confirmed?).to eq true
    end
    it 'undo confirmation of user' do
      patch :toggle_confirmation, id: user.id, user: { to_confirm: 'false' }
      expect(user.confirmed?).to eq false
    end
  end
```
This test keeps on failing. I guess it's my unfamiliarity with RSpec that's causing the problem. What am I missing here?